samples/pg_to_alloy_migration_search.ipynb

```diff
@@ -0,0 +1,488 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "HPcWWr_LaLqD"
+   },
+   "outputs": [],
+   "source": [
+    "# @title Set Your Values Here { display-mode: \"form\" }\n",
+    "USER = \"postgres\"  # @param {type: \"string\"}\n",
+    "PASSWORD = input(f\"Please provide a password to be used for {USER} database user: \")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "5_-kcRf4bJ8B"
+   },
+   "source": [
+    "Create a connection to your AlloyDB for PostgreSQL instance using the AlloyDBEngine class.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "rWlcGt3JIDIy"
+   },
+   "outputs": [],
+   "source": [
+    "from langchain_google_alloydb_pg import AlloyDBEngine, AlloyDBVectorStore\n",
+    "from langchain_core.documents import Document\n",
+    "\n",
+    "engine = AlloyDBEngine.from_instance(\n",
+    "    project_id=PROJECT_ID,\n",
+    "    region=REGION,\n",
+    "    cluster=CLUSTER,\n",
+    "    instance=INSTANCE,\n",
+    "    database=DATABASE,\n",
+    "    user=USER,\n",
+    "    password=PASSWORD,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "qI3IdMdubu4y"
+   },
+   "source": [
+    "### Create an embedding class instance\n",
+    "\n",
+    "You can use any [LangChain embeddings model](https://python.langchain.com/docs/integrations/text_embedding/).\n",
+    "You may need to enable Vertex AI API to use `VertexAIEmbeddings`. We recommend setting the embedding model's version for production, learn more about the [Text embeddings models](https://cloud.google.com/vertex-ai/docs/generative-ai/model-reference/text-embeddings)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "vMeZsVuLb_6g"
+   },
+   "outputs": [],
+   "source": [
+    "# enable Vertex AI API\n",
+    "!gcloud services enable aiplatform.googleapis.com"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "69OOOCWebc5X"
+   },
+   "outputs": [],
+   "source": [
+    "from langchain_google_vertexai import VertexAIEmbeddings\n",
+    "\n",
+    "embeddings_service = VertexAIEmbeddings(\n",
+    "    model_name=\"textembedding-gecko@003\", project=PROJECT_ID\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "iY8oA59vcL7_"
+   },
+   "source": [
+    "### Initialize an AlloyDB Loader\n",
+    "\n",
+    "Intialize an AlloyDB Loader to fetch collection uuid from the \"langchain_pg_collection\" table"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "phMW5OS1IOA9"
+   },
+   "outputs": [],
+   "source": [
+    "from langchain_google_alloydb_pg import AlloyDBLoader\n",
+    "\n",
+    "# Choose the collection to perform vector search on\n",
```

Comment:
Please provide the context on how PGVector is setup so the user can follow the train of thought that we don't want to just search the whole vector store since all of the embeddings are in the same table